File: Exercise/2-simpleArraySum.js
// arr = [1,2,3] -> Find the sum of its elements (1+2+3)
// return 6


function simpleArraySum(ar){
  let sum = 0;
  for(let i = 0; i < ar.length; i++){
    sum += ar[i];
  }
  
  return sum;
}

const arr = [3,2,3];
const result = simpleArraySum(arr);
console.log(result);

File: Exercise PY/2-simpleArraySum.py
# arr = [1,2,3] -> Find the sum of its elements (1+2+3)
# return 6

def simpleArrSum(arr):
  sum = 0
  for i in arr:
    sum += i
  
  return sum
    


arr = [1, 2, 3]
result = simpleArrSum(arr)
print({'Array sum result is': result})
